src/SpectatorView.Unity/Assets/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/ArUcoMarkerDetector.cs

```diff
@@ -84,13 +88,10 @@ public MarkerPositionBehavior MarkerPositionBehavior
             }
         }
 
-        private void Awake()
+        protected void OnEnable()

```

Comment:
>protected [](start = 8, length = 9)

I think these could all be consistently private. If it was expected that someone might override this class, all of the MonoBehaviour methods should be "protected virtual"